build/dart/vochain/vochain.pb.dart

```diff
@@ -813,6 +813,7 @@ class ProofArbo extends $pb.GeneratedMessage {
     ..a<$core.List<$core.int>>(2, const $core.bool.fromEnvironment('protobuf.omit_field_names') ? '' : 'siblings', $pb.PbFieldType.OY)
     ..a<$core.List<$core.int>>(3, const $core.bool.fromEnvironment('protobuf.omit_field_names') ? '' : 'value', $pb.PbFieldType.OY)
     ..e<ProofArbo_KeyType>(4, const $core.bool.fromEnvironment('protobuf.omit_field_names') ? '' : 'keyType', $pb.PbFieldType.OE, protoName: 'keyType', defaultOrMaker: ProofArbo_KeyType.PUBKEY, valueOf: ProofArbo_KeyType.valueOf, enumValues: ProofArbo_KeyType.values)
+    ..a<$core.List<$core.int>>(5, const $core.bool.fromEnvironment('protobuf.omit_field_names') ? '' : 'weight', $pb.PbFieldType.OY)
```

Comment:
You should not commit the protobuf generated files, only the `src/` directoy. There is a CI configured for generating the bindings automatically.